app/src/main/res/values/strings.xml

```diff
@@ -384,4 +384,5 @@
     <string name="not_enrolled_policies">Not Enrolled Policies</string>
     <string name="request_enrolled_policies">Request for Not Enrolled Policies</string>
     <string name="phone_number_not_provided">Phone number not provided</string>
+    <string name="not_unique_products">Insurance products of the selected policies are not unique</string>
```

Comment:
Can you change this with 'Les produits d'assurance des polices sélectionnées ne sont pas uniques'? Thanks.